### ISSUE TITLE
FAILED - RETRYING: Install Docker atomic-1.22.1-25.git5a342e3.el7.centos.x86_64

### ISSUE THREAD

**Issue by piyushkv1** [openshift/openshift-ansible#10702]
#### Description
ansible-playbook -i nsx-integration-for-openshift/openshift-ansible-nsx/hosts openshift-ansible/playbooks/prerequisites.yml

TASK [container_runtime : Install Docker] ************************************************************************************************************************************************
FAILED - RETRYING: Install Docker (3 retries left).
FAILED - RETRYING: Install Docker (3 retries left).
FAILED - RETRYING: Install Docker (2 retries left).
FAILED - RETRYING: Install Docker (2 retries left).
FAILED - RETRYING: Install Docker (1 retries left).
FAILED - RETRYING: Install Docker (1 retries left).
fatal: [fvt1796-250-135.eng.vmware.com]: FAILED! => {"attempts": 3, "changed": false, "msg": "Error: Package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64 (extras)\n           Requires: python-docker-py >= 1.7.2-1\n           Available: docker-python-1.4.0-115.el7.x86_64 (extras)\n               python-docker-py = 1.4.0-115.el7\n           Available: python-docker-py-1.10.6-3.el7.noarch (extras)\n               python-docker-py = 1.10.6-3.el7\n           Available: python-docker-py-1.10.6-4.el7.noarch (extras)\n               python-docker-py = 1.10.6-4.el7\n", "rc": 1, "results": ["Loaded plugins: fastestmirror\nLoading mirror speeds from cached hostfile\n * base: mirror.den1.denvercolo.net\n * epel: mirror.sjc02.svwh.net\n * extras: mirror.den1.denvercolo.net\n * updates: mirror.den1.denvercolo.net\nResolving Dependencies\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: skopeo-containers >= 1:0.1.29-3 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n--> Processing Dependency: runc for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: python-dateutil for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: ostree for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: gomtree for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: container-storage-setup for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n---> Package docker.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n--> Processing Dependency: docker-common = 2:1.13.1-75.git8633870.el7.centos for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: docker-client = 2:1.13.1-75.git8633870.el7.centos for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: subscription-manager-rhsm-certificates for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n---> Package skopeo.x86_64 1:0.1.31-1.dev.gitae64ff7.el7.centos will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package container-storage-setup.noarch 0:0.11.0-2.git5eaf76c.el7 will be installed\n---> Package docker-client.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n---> Package docker-common.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n--> Processing Dependency: oci-umount >= 2:2.3.3-3 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: oci-systemd-hook >= 1:0.1.4-9 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: oci-register-machine >= 1:0-5.13 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: container-selinux >= 2:2.51-1 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: atomic-registries for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n---> Package gomtree.x86_64 0:0.5.0-0.2.git16da0f8.el7 will be installed\n---> Package ostree.x86_64 0:2018.5-1.el7 will be installed\n--> Processing Dependency: libarchive.so.13()(64bit) for package: ostree-2018.5-1.el7.x86_64\n---> Package python-dateutil.noarch 0:1.5-7.el7 will be installed\n---> Package runc.x86_64 0:1.0.0-52.dev.git70ca035.el7_5 will be installed\n--> Processing Dependency: criu for package: runc-1.0.0-52.dev.git70ca035.el7_5.x86_64\n---> Package skopeo-containers.x86_64 1:0.1.31-1.dev.gitae64ff7.el7.centos will be installed\n---> Package subscription-manager-rhsm-certificates.x86_64 0:1.20.11-1.el7.centos will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package atomic-registries.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-pytoml for package: 1:atomic-registries-1.22.1-25.git5a342e3.el7.centos.x86_64\n---> Package container-selinux.noarch 2:2.68-1.el7 will be installed\n--> Processing Dependency: policycoreutils-python for package: 2:container-selinux-2.68-1.el7.noarch\n---> Package criu.x86_64 0:3.5-4.el7 will be installed\n--> Processing Dependency: libprotobuf-c.so.1(LIBPROTOBUF_C_1.0.0)(64bit) for package: criu-3.5-4.el7.x86_64\n--> Processing Dependency: libprotobuf-c.so.1()(64bit) for package: criu-3.5-4.el7.x86_64\n--> Processing Dependency: libnet.so.1()(64bit) for package: criu-3.5-4.el7.x86_64\n---> Package libarchive.x86_64 0:3.1.2-10.el7_2 will be installed\n---> Package oci-register-machine.x86_64 1:0-6.git2b44233.el7 will be installed\n---> Package oci-systemd-hook.x86_64 1:0.1.17-2.git83283a0.el7 will be installed\n--> Processing Dependency: libyajl.so.2()(64bit) for package: 1:oci-systemd-hook-0.1.17-2.git83283a0.el7.x86_64\n---> Package oci-umount.x86_64 2:2.3.3-3.gite3c9055.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package libnet.x86_64 0:1.1.6-7.el7 will be installed\n---> Package policycoreutils-python.x86_64 0:2.5-22.el7 will be installed\n--> Processing Dependency: setools-libs >= 3.3.8-2 for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: audit-libs-python >= 2.1.3-4 for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: python-IPy for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1(VERS_1.4)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1(VERS_1.2)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libcgroup for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libapol.so.4(VERS_4.0)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: checkpolicy for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1()(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libapol.so.4()(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n---> Package protobuf-c.x86_64 0:1.0.2-3.el7 will be installed\n---> Package python-pytoml.noarch 0:0.1.14-1.git7dea353.el7 will be installed\n---> Package yajl.x86_64 0:2.0.4-4.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit-libs-python.x86_64 0:2.8.1-3.el7_5.1 will be installed\n--> Processing Dependency: audit-libs(x86-64) = 2.8.1-3.el7_5.1 for package: audit-libs-python-2.8.1-3.el7_5.1.x86_64\n---> Package checkpolicy.x86_64 0:2.5-6.el7 will be installed\n---> Package libcgroup.x86_64 0:0.41-15.el7 will be installed\n---> Package python-IPy.noarch 0:0.75-6.el7 will be installed\n---> Package setools-libs.x86_64 0:3.3.8-2.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit-libs.x86_64 0:2.8.1-3.el7 will be updated\n--> Processing Dependency: audit-libs(x86-64) = 2.8.1-3.el7 for package: audit-2.8.1-3.el7.x86_64\n---> Package audit-libs.x86_64 0:2.8.1-3.el7_5.1 will be an update\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit.x86_64 0:2.8.1-3.el7 will be updated\n---> Package audit.x86_64 0:2.8.1-3.el7_5.1 will be an update\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
fatal: [fvt1796-250-133.eng.vmware.com]: FAILED! => {"attempts": 3, "changed": false, "msg": "Error: Package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64 (extras)\n           Requires: python-docker-py >= 1.7.2-1\n           Available: docker-python-1.4.0-115.el7.x86_64 (extras)\n               python-docker-py = 1.4.0-115.el7\n           Available: python-docker-py-1.10.6-3.el7.noarch (extras)\n               python-docker-py = 1.10.6-3.el7\n           Available: python-docker-py-1.10.6-4.el7.noarch (extras)\n               python-docker-py = 1.10.6-4.el7\n", "rc": 1, "results": ["Loaded plugins: fastestmirror\nLoading mirror speeds from cached hostfile\n * base: centos.den.host-engine.com\n * epel: d2lzkl7pfhq30w.cloudfront.net\n * extras: mirrors.syringanetworks.net\n * updates: mirrors.ocf.berkeley.edu\nResolving Dependencies\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: skopeo-containers >= 1:0.1.29-3 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n--> Processing Dependency: runc for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: python-dateutil for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: ostree for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: gomtree for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n--> Processing Dependency: container-storage-setup for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\n---> Package docker.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n--> Processing Dependency: docker-common = 2:1.13.1-75.git8633870.el7.centos for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: docker-client = 2:1.13.1-75.git8633870.el7.centos for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: subscription-manager-rhsm-certificates for package: 2:docker-1.13.1-75.git8633870.el7.centos.x86_64\n---> Package skopeo.x86_64 1:0.1.31-1.dev.gitae64ff7.el7.centos will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package container-storage-setup.noarch 0:0.11.0-2.git5eaf76c.el7 will be installed\n---> Package docker-client.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n---> Package docker-common.x86_64 2:1.13.1-75.git8633870.el7.centos will be installed\n--> Processing Dependency: oci-umount >= 2:2.3.3-3 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: oci-systemd-hook >= 1:0.1.4-9 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: oci-register-machine >= 1:0-5.13 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: container-selinux >= 2:2.51-1 for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n--> Processing Dependency: atomic-registries for package: 2:docker-common-1.13.1-75.git8633870.el7.centos.x86_64\n---> Package gomtree.x86_64 0:0.5.0-0.2.git16da0f8.el7 will be installed\n---> Package ostree.x86_64 0:2018.5-1.el7 will be installed\n--> Processing Dependency: libarchive.so.13()(64bit) for package: ostree-2018.5-1.el7.x86_64\n---> Package python-dateutil.noarch 0:1.5-7.el7 will be installed\n---> Package runc.x86_64 0:1.0.0-52.dev.git70ca035.el7_5 will be installed\n--> Processing Dependency: criu for package: runc-1.0.0-52.dev.git70ca035.el7_5.x86_64\n---> Package skopeo-containers.x86_64 1:0.1.31-1.dev.gitae64ff7.el7.centos will be installed\n---> Package subscription-manager-rhsm-certificates.x86_64 0:1.20.11-1.el7.centos will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package atomic-registries.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-pytoml for package: 1:atomic-registries-1.22.1-25.git5a342e3.el7.centos.x86_64\n---> Package container-selinux.noarch 2:2.68-1.el7 will be installed\n--> Processing Dependency: policycoreutils-python for package: 2:container-selinux-2.68-1.el7.noarch\n---> Package criu.x86_64 0:3.5-4.el7 will be installed\n--> Processing Dependency: libprotobuf-c.so.1(LIBPROTOBUF_C_1.0.0)(64bit) for package: criu-3.5-4.el7.x86_64\n--> Processing Dependency: libprotobuf-c.so.1()(64bit) for package: criu-3.5-4.el7.x86_64\n--> Processing Dependency: libnet.so.1()(64bit) for package: criu-3.5-4.el7.x86_64\n---> Package libarchive.x86_64 0:3.1.2-10.el7_2 will be installed\n---> Package oci-register-machine.x86_64 1:0-6.git2b44233.el7 will be installed\n---> Package oci-systemd-hook.x86_64 1:0.1.17-2.git83283a0.el7 will be installed\n--> Processing Dependency: libyajl.so.2()(64bit) for package: 1:oci-systemd-hook-0.1.17-2.git83283a0.el7.x86_64\n---> Package oci-umount.x86_64 2:2.3.3-3.gite3c9055.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package libnet.x86_64 0:1.1.6-7.el7 will be installed\n---> Package policycoreutils-python.x86_64 0:2.5-22.el7 will be installed\n--> Processing Dependency: setools-libs >= 3.3.8-2 for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: audit-libs-python >= 2.1.3-4 for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: python-IPy for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1(VERS_1.4)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1(VERS_1.2)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libcgroup for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libapol.so.4(VERS_4.0)(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: checkpolicy for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libqpol.so.1()(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n--> Processing Dependency: libapol.so.4()(64bit) for package: policycoreutils-python-2.5-22.el7.x86_64\n---> Package protobuf-c.x86_64 0:1.0.2-3.el7 will be installed\n---> Package python-pytoml.noarch 0:0.1.14-1.git7dea353.el7 will be installed\n---> Package yajl.x86_64 0:2.0.4-4.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit-libs-python.x86_64 0:2.8.1-3.el7_5.1 will be installed\n--> Processing Dependency: audit-libs(x86-64) = 2.8.1-3.el7_5.1 for package: audit-libs-python-2.8.1-3.el7_5.1.x86_64\n---> Package checkpolicy.x86_64 0:2.5-6.el7 will be installed\n---> Package libcgroup.x86_64 0:0.41-15.el7 will be installed\n---> Package python-IPy.noarch 0:0.75-6.el7 will be installed\n---> Package setools-libs.x86_64 0:3.3.8-2.el7 will be installed\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit-libs.x86_64 0:2.8.1-3.el7 will be updated\n--> Processing Dependency: audit-libs(x86-64) = 2.8.1-3.el7 for package: audit-2.8.1-3.el7.x86_64\n---> Package audit-libs.x86_64 0:2.8.1-3.el7_5.1 will be an update\n--> Running transaction check\n---> Package atomic.x86_64 1:1.22.1-25.git5a342e3.el7.centos will be installed\n--> Processing Dependency: python-docker-py >= 1.7.2-1 for package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64\nPackage python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed\n---> Package audit.x86_64 0:2.8.1-3.el7 will be updated\n---> Package audit.x86_64 0:2.8.1-3.el7_5.1 will be an update\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
	to retry, use: --limit @/root/openshift-ansible/playbooks/prerequisites.retry

PLAY RECAP *******************************************************************************************************************************************************************************
fvt1796-250-133.eng.vmware.com : ok=63   changed=7    unreachable=0    failed=1
fvt1796-250-135.eng.vmware.com : ok=43   changed=7    unreachable=0    failed=1
localhost                  : ok=11   changed=0    unreachable=0    failed=0

--> Finished Dependency Resolution
Error: Package: 1:atomic-1.22.1-25.git5a342e3.el7.centos.x86_64 (extras)
           Requires: python-docker-py >= 1.7.2-1
           Available: docker-python-1.4.0-115.el7.x86_64 (extras)
               python-docker-py = 1.4.0-115.el7
           Available: python-docker-py-1.10.6-3.el7.noarch (extras)
               python-docker-py = 1.10.6-3.el7
           Available: python-docker-py-1.10.6-4.el7.noarch (extras)
               python-docker-py = 1.10.6-4.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@fvt1796-250-135 ~]# yum install python-docker-py-1.10.6-4.el7.noarch
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.den1.denvercolo.net
 * epel: mirror.sjc02.svwh.net
 * extras: mirror.den1.denvercolo.net
 * updates: mirror.den1.denvercolo.net
Package python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed
Nothing to do

I tried installation manually.
[root@fvt1796-250-133 ~]# yum install python-docker-py-1.10.6-4.el7.noarch
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.den.host-engine.com
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirrors.syringanetworks.net
 * updates: mirrors.ocf.berkeley.edu
Package python-docker-py-1.10.6-4.el7.noarch is obsoleted by python2-docker-3.3.0-1.el7.noarch which is already installed
Nothing to do
[root@fvt1796-250-133 ~]# yum install python2-docker-3.3.0-1.el7.noarch
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.den.host-engine.com
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirrors.syringanetworks.net
 * updates: mirrors.ocf.berkeley.edu
Package python2-docker-3.3.0-1.el7.noarch already installed and latest version
Nothing to do

##### Version

Please put the following version information in the code block
indicated below.

* Your ansible version per `ansible --version`

[root@fvt1796-250-133 ~]# ansible --version
ansible 2.7.1
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr 11 2018, 07:36:10) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

* The output of `git describe`

[root@fvt1796-250-133 openshift-ansible]# git describe
openshift-ansible-3.11.44-1-2-geae45eb

* The output of `rpm -q openshift-ansible`

[root@fvt1796-250-133 openshift-ansible]# rpm -q openshift-ansible
package openshift-ansible is not installed

**Comment by DanyC97** [comment]
few things before we can even look into it:

* _please, please_ format the error output as a code, is very hard to follow what you dumped. 
* can i see the output (again formatted) of your `/etc/yum.repos.d/` and the content of each repo file ? (you could even use gits.github or anything else rather than dump it here, whatever works
* your starting point, did you have any docker version installed prior to run the installation ?

**Comment by billyfoss** [comment]
I ran into similar output today.  I discovered it was a problem with the generic Origin repo installing based on openshift_version which is not defined until after the repos are installed. I was running into missing 3.11 packages and was also trying several testing repos. 

I found it related to https://github.com/openshift/openshift-ansible/issues/10445 and https://github.com/openshift/openshift-ansible/pull/10640. 

I got past it by removing the Origin repos, manually removing python2-docker and explicitly setting openshift_version=3.11 in my inventory.  Then the prerequistes.yml completed.
